mern-ecommerce/server.js

```diff
@@ -49,7 +49,7 @@ app.use("/api", userRoutes);
 app.use("/api", categoryRoutes);
 app.use("/api", productRoutes);
 app.use("/api", braintreeRoutes);
-app.use("/api", orderRoutes);
+app.use("/api", orderRoute);
```

Comment:
The variable name `orderRoute` should be changed to `orderRoutes` to match the actual file name and convention used in the codebase.